### ISSUE TITLE
#110 Reincarnated

### ISSUE THREAD

**Issue by tyler2k15** [TCPR/Thermos#265]
**Explanation of issue:**
https://github.com/TCPR/Thermos/issues/110 Issue 110 

**How to recreate this issue:**
Try using Autonomous Activator. Does not perform as usual. (Using Towny as Protection) I believe this is what caused the issue to return: https://github.com/TCPR/Thermos/commit/3ca312388a14127cc6e99d734c2767c34084799e
https://github.com/TCPR/Thermos/compare/48...49

**Modpack Name:** *(Only if public)*
Multiple Modpacks - i.e. RR3 Mainline

**Plugins Installed:**
Towny for protection + others

**Thermos Version:**
Any builds after 48 (49+)

**Forge Version:**
1614

**Comment by tyler2k15** [comment]
I have received reports this is still not solved in 54

**Comment by tyler2k15** [comment]
@Robotia

**Comment by Robotia** [comment]
Ok I will try to see what's wrong

**Comment by KasperFranz** [comment]
This is still present in 57

**Comment by kristi71111** [comment]
Can also confirm that issue is still present.

**Comment by Time6628** [comment]
Does this only happen with Towny?

**Comment by KasperFranz** [comment]
We are using Grief prevention - but it is also a problem outside of claims.

**Comment by Time6628** [comment]
With whatever permission plugin you are using, give `5ae51d0b-e8bc-5a02-09f4-b5dbb05963da` the `*` permission. That uuid is the uuid of the fake player for CoFH.

**Comment by KasperFranz** [comment]
this is still present in latest build

**Comment by Robotia** [comment]
We thought it was fake player bug but on Frani's server it was auth plugin causing the problem

**Comment by Eufranio** [comment]
Yes, i had AuthMe (if you guys use AuthMe you need to put the Fake Player's name in the config).
